jboss-eap-clustered-vmss-rhel7/README.md

```diff
@@ -48,28 +47,29 @@ Following is the Architecture:
 
 ![alt text](images/arch.png)
 
-To learn more about the JBoss Enterprise Application Platform, visit:
-https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/
+To learn more about the JBoss Enterprise Application Platform, visit [Red Hat EAP 7.2 documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/).
```

Comment:
The change mentioned in “Template Solution Architecture” was moved under “Solution Architecture” in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here